accounts-db/src/accounts_index.rs

```diff
@@ -73,8 +73,9 @@ pub const ACCOUNTS_INDEX_CONFIG_FOR_BENCHMARKS: AccountsIndexConfig = AccountsIn
     num_initial_accounts: None,
 };
 pub type ScanResult<T> = Result<T, ScanError>;
-pub type SlotList<T> = SmallVec<[(Slot, T); 1]>;
-pub type ReclaimsSlotList<T> = Vec<(Slot, T)>;
+pub type SlotList<T> = SmallVec<[SlotListItem<T>; 1]>;
+pub type ReclaimsSlotList<T> = Vec<SlotListItem<T>>;
+pub type SlotListItem<T> = (Slot, T);
```

Comment:
Here's the new alias. I didn't want to go with `Entry` in the name, since we also have the `AccountMapEntry` (and also `Entry` is a different third thing in std Containers like HashMap). Since `Item` is used as the associate type in many rust std containers/iterators, I went with that.